README.md

```diff
@@ -1,214 +1,214 @@
+- [Try out Label Studio](#try-out-label-studio)
+- [What you get from Label Studio](#what-you-get-from-label-studio)
+- [Included templates for labeling data in Label Studio](#included-templates-for-labeling-data-in-label-studio)
+- [Integrate with machine learning models](#integrate-with-machine-learning-models)
+- [Integrate Label Studio with your machine learning tools](#integrate-label-studio-with-your-machine-learning-tools)
 
-## Summary
+![Gif of Label Studio annotating different types of data](https://raw.githubusercontent.com/heartexlabs/label-studio/master/images/new_annotation_examples.gif)
 
-<img align="right" height="180" src="https://github.com/heartexlabs/label-studio/blob/master/images/heartex_icon_opossum_green@2x.png?raw=true" />
+Have a custom dataset? You can customize Label Studio to fit your needs. Read an [introductory blog post](https://towardsdatascience.com/introducing-label-studio-a-swiss-army-knife-of-data-labeling-140c1be92881#3907-fd502dc24c8d) to learn more. 
 
-- [Quick Start](#quick-start)
-- [One Click Deploy](#one-click-deploy)
-- [Features :star2:](#features-star2)
-- [Use Cases](#use-cases)
-- [Machine Learning Integration](#machine-learning-integration)
-- [For Teams and Enterprises :office:](#label-studio-for-teams-startups-and-enterprises-office)
-- [Ecosystem](#ecosystem)
-- [License](#license)
+## Try out Label Studio
 
-## Quick Start
+Try out Label Studio in a **[running app](https://app.labelstud.io)**, install it locally, or deploy it in a cloud instance. 
 
-```bash
-# Requires >=Python3.5, Python 3.9 is not supported
-pip install -U label-studio
+- [Install locally with Docker](#install-locally-with-docker)
+- [Run with Docker Compose](#run-with-docker-compose)
+- [Install locally with pip](#install-locally-with-pip)
+- [Install locally with Anaconda](#install-locally-with-anaconda)
+- [Install for local development](#install-for-local-development)
+- [Deploy in a cloud instance](#deploy-in-a-cloud-instance)
 
-# Initialize the project in labeling_project path
-label-studio init labeling_project
+### Install locally with Docker
+Run Label Studio in a Docker container and access it at `http://localhost:8080`.
 
-# Start the server at http://localhost:8080
-label-studio start labeling_project
+```bash
+docker run --rm -p 8080:8080 -v `pwd`/my_project:/label-studio/my_project --name label-studio heartexlabs/label-studio:latest label-studio start my_project --init
 ```
 
-#### Install on Windows
-
-For running on Windows, the following wheel packages are needed to be manually downloaded from [Gohlke builds](https://www.lfd.uci.edu/~gohlke/pythonlibs), by ensuring the right python version:
-
-- [lxml](https://www.lfd.uci.edu/~gohlke/pythonlibs/#lxml)
+By default, this command creates a blank project in a `./my_project` directory. If the `./my_project` folder already exists, Label Studio fails to start. Rename or delete the folder, or use the `--force` argument to force Label Studio to start: 
```

Comment:
What about adding `#### Overriding default docker command` here? It is frequently asked, and it would be nice to have an anchor link for that

Reply:
done